code/modules/mob/living/simple_animal/vore/solargrub.dm

```diff
@@ -96,6 +96,8 @@ List of things solar grubs should be able to do:
 			PN = null
 
 /mob/living/simple_animal/retaliate/solargrub //active noms
+	var/vore_bump_chance = 50
+	var/vore_bump_emote = "tries to slurp you up with minimal effort."	
```

Comment:
"the solargrub tries to slurp you up with minimal effort Urist McPrey!"

The displayed text is of the form [the pred] [vore bump emote] [the prey].